src/data/template/Microsoft.Authorization/policyAssignments.template.jq

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {},
+    "resources": [
+        {
+            "type": .ResourceType,
+            "name": .Name,
+            "apiVersion": "0000-00-00",
+            "location": "[deployment().location]",
```

Comment:
location is only needed when system MI is required for DINE/Modify

Reply:
can we keep it then as we dont know whether it is is DINE policy assignment or not?

Reply:
Yes